apps/condo/domains/common/components/TicketCard/TicketCardList.tsx

```diff
@@ -1,19 +1,21 @@
-import React, { useMemo } from 'react'
+import { green } from '@ant-design/colors'
+import { SortTicketsBy, Ticket as TicketSchema } from '@app/condo/schema'
+import { Loader } from '@condo/domains/common/components/Loader'
+import { colors } from '@condo/domains/common/constants/style'
+import { Ticket } from '@condo/domains/ticket/utils/clientSchema'
 import { useIntl } from '@core/next/intl'
-import Link from 'next/link'
-import qs, { IStringifyOptions } from 'qs'
-import { Row, Col, Typography, RowProps, Space } from 'antd'
+import styled from '@emotion/styled'
+import { Col, Row, RowProps, Space, Typography } from 'antd'
 import get from 'lodash/get'
-import pickBy from 'lodash/pickBy'
 import groupBy from 'lodash/groupBy'
-import styled from '@emotion/styled'
-import { colors } from '@condo/domains/common/constants/style'
-import { Ticket } from '@condo/domains/ticket/utils/clientSchema'
-import { SortTicketsBy, Ticket as TicketSchema } from '@app/condo/schema'
-import { Loader } from '@condo/domains/common/components/Loader'
-import { TicketOverview } from './TicketOverview'
-import { green } from '@ant-design/colors'
+import pickBy from 'lodash/pickBy'
+import Link from 'next/link'
+import qs, { IStringifyOptions } from 'qs'
+import React, { useEffect, useMemo } from 'react'
+import { Ticket as TicketGQL } from '../../../ticket/gql'
```

Comment:
`@condo/domains/ticket`
```suggestion
import { Ticket as TicketGQL } from '@condo/domains/ticket/gql'
```

Reply:
Why do you even need this import?)

Reply:
Fixed. Unused imports removed.